README.md

```diff
@@ -303,16 +299,15 @@ Just follow 3 rules:
     composer fix-cs
     ```
 
-We would be happy to merge your feature then.
+We would be happy to accept PRs that follow these guidelines.
 
 ## Run Rector in Docker
-
-With this command, you can process your project with Rector from docker:
+You can run Rector on your project using Docker:
 
 ```bash
 docker run -v $(pwd):/project rector/rector:latest process /project/src --set symfony40 --dry-run
 
-# Note that a volume is mounted from `pwd` into `/project` which can be accessed later.
+# Note that a volume is mounted from the current directory into `/project` which can be accessed later.
```

Comment:
Better combine both, as one is more technical and other amateur (so both groups understand)